httpdocs/css/webots-cloud.css

```
@@ -6,11 +6,11 @@ body {
   height: 100%;
 }
 body:after{
-  content: "beta version";
+  content: "testing";
```

```suggestion
  content: "beta version";
```

As you will merge it in beta